clients.py

```diff
@@ -0,0 +1,59 @@
+CLIENTS_MENU = """
+1. Add Client
+2. Remove Client
+3. Update client
+4. Display All Clients
+X. Go Back To Main Menu
+"""
+
+
+clients = {
+    1: {'last_name': 'Pop',
+        'first_name': 'Ion',
+        'date_of_birth': '21/08/1968',
+        'email': 'popion@email.com'
+        },
+    2: {
+        'last_name': 'Stan',
+        'first_name': 'Ana',
+        'date_of_birt': '14/06/2001',
+        'email': 'stanana@email.com'
+    }
+}
+
+
+def add_client():
+    # check if the id is used, if yes select another one
+    client_id = int(input('Please provide a client_id: '))
```

Comment:
We could have this in a while in order to ask for a different id in case it's been used. Now, the comment above is confusing